src/views/tutorial-view/components/mdx-img/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface MdxImgProps {
+  /** Source attribute for the image tag. */
+  src: JSX.IntrinsicElements['img']['src']
+  /** Alternate text for the image tag. Should be a text equivalent to the information conveyed in the image. As an example, the alternate text "Diagram of Humans or Machines Clients" is likely not sufficient because it does not express all of the same information as the diagram does for sighted users. It's important to provide the same information experience for all users regardless of ability. */
```

Comment:
Tried to add some descriptive comments here to match the work done in the Figma file 👍